includes/gateway/class-omise-payment-mobilebanking.php

```diff
@@ -81,7 +81,10 @@ public function charge( $order_id, $order ) {
 		);
 
 		$source_type = sanitize_text_field( $_POST['omise-offsite']);
-		$return_uri = add_query_arg('order_id', $order_id, home_url('wc-api/omise_mobilebanking_callback'));
```

Comment:
 so the root cause it bc we can't consturct by using this format right? 
 then why we roll back to 4.22.2 and it work na 🤔